scss/_footer.scss

```diff
@@ -28,8 +28,12 @@
       .nav-link {
         outline-offset: $focus-visible-outer-offset; // Override outline offset coming from .navbar-nav
 
-        &.active::before {
-          content: unset;
+        &.active {
```

Comment:
For demonstration purpose only.
This whole part can be removed as soon as #1859 is merged.